content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -0,0 +1,139 @@
+=================
+Calls for Tenders
+=================
+
+**Calls for Tenders** are one of the two main types of **Purchase Agreements** available by default
+in Odoo.
+
+*Calls for Tenders* are used to request offers from multiple vendors at the same time, by inviting
+those vendors to submit offers for similar goods or services all at once. This helps companies that
+use this feature to select the cheapest and fastest vendors in the market, depending on their
+needs.
+
+While calls for tenders are primaily used by organizations in the public sector who are legally
```

Comment:
Calls for Tenders...like the other features I've commented on above, should look uniform throughout the doc - and they should stand out in a sentence, to make things a bit clearer for the reader. So, either capitalize it throughout, toss some * on either side, hit it with a :guilabel: ... someting like that. What do you think?

Reply:
...in the public sector, who are legally...

Reply:
I definitely agree with you on making it uniform and stand our throughout the doc. After first mention, I've decided to capitalize.